sidebars.js

```diff
@@ -64,7 +64,8 @@ module.exports = {
       items: [
         'mathematics/mathematics',
         'mathematics/armstrong-number',
-	  'mathematics/factorial-of-number'
+	'mathematics/factorial-of-number',
+        'mathematics/happy-number'
```

Comment:
Indent and format it

Reply:
@HarshCasper actually, in my system, it is showing properly but here it is showing like this. Can you correct it as I am not able to if you can. Thank you.